### PR TITLE
Added tests for Student object and routes

### PR DESCRIPTION
**Pull request by Krishna-Sivakumar**
Addresses Issue #13

- Added test for Student object
- Added test for routes

Please review my code and comment if any other tests could be added. Thanks!

### CODE REVIEW

**Comment by marceloFA** [comment]
Great work! Would you describe on the readme.md how one can run the app tests? @Krishna-Sivakumar

**Comment by Krishna-Sivakumar** [comment]
Okay, shall do

File: users/tests.py [hunk 1]
```diff
@@ -51,4 +50,4 @@ def test_for_response(self):
         routes = ['/logout/']
         for i in routes:
```
[the remark this answers is not in the document]
Comment (Krishna-Sivakumar) [reply users/tests.py]: Okay

**Comment by marceloFA** [comment]
Super! Thanks again

**Pull request merged**